pkg/platform/api/buildplanner/model/buildplan.go

```diff
@@ -310,12 +310,11 @@ func ProcessBuildError(build *Build, fallbackMessage string) error {
 }
 
 func ProcessProjectError(project *Project, fallbackMessage string) error {
-	if project.Error == nil {
-		return errs.New(fallbackMessage)
-	}
```

Comment:
The change here is like the one here: https://github.com/ActiveState/cli/pull/2743/files#diff-14612946db10f18744a68890b9e353cf87d4da9104592d9fe4346264ed83ef77L281-L284